leases/utils.py

```diff
@@ -13,7 +13,9 @@
 
 from berth_reservations.exceptions import VenepaikkaGraphQLError
 from customers.services import ProfileService
+from leases.consts import TERMINABLE_STATUSES
 from leases.enums import LeaseStatus
```

Comment:
Can you also replace this with relative imports? 😇 
```suggestion
from .consts import TERMINABLE_STATUSES
from .enums import LeaseStatus
```

Reply:
fixed